To-do-List.js

```diff
@@ -0,0 +1,232 @@
+                checkbox.checked = true;
+            }
+        
+            p.id = `p${arr1[i]}`;
+            p.innerText = items[arr1[i]].text;
+        
+            delButton.type = 'button';
+            delButton.className = 'button';
+            var del = document.createElement('i');
+            del.className = 'fa fa-trash';
+            delButton.appendChild(del);
+            delButton.id = `d${arr1[i]}`;
+            delButton.addEventListener("click", deleteItem);
+            
+            list.id = `${arr1[i]}`;
+            list.appendChild(checkbox);
+            list.appendChild(p);
+            list.appendChild(delButton);
+        
+            div.className = 'downAnim';
+            div.appendChild(list);
+        
+            appendItems.appendChild(div);
+        }        
+    }
+
+    else {
+        for (let i = 0; i< items.length; i++) {
+            newList(i);
+            sort();
+        }
+        document.getElementById('appendItem').lastChild.lastChild.className = 'addAnim';
+    }
+    
+    setTimeout(() => {
+        sort();
+    }, 400);        
+};
+
+let sort = () => {
+    var appendItems = document.getElementById('appendItem');
+    var toSort = document.getElementById('sortList').options.selectedIndex;
+
+    if (toSort === 1) {
+        appendItems.innerHTML = '';
+
+        for (let i = 0; i< items.length; i++) {
+            if (items[i].status == 'checked') {
+                newList(i);
+            }
+        }
+
+        for (let i = 0; i< items.length; i++) {
+            if (items[i].status == 'unchecked') {
+                newList(i);
+            }
+        }
+    }
+
+    else if (toSort === 2) {
+        appendItems.innerHTML = '';
+
+        for (let i = 0; i< items.length; i++) {
+            if (items[i].status == 'unchecked') {
+                newList(i);
+            }
+        }
+
+        for (let i = 0; i< items.length; i++) {
+            if (items[i].status == 'checked') {
+                newList(i);
+            }
+        }
+    }
+
+    else if (toSort === 3) {
+        appendItems.innerHTML = '';
+        for (let i = 0; i< itemsCopy.length; i++) {
+            var div = document.createElement('div');
+            var list = document.createElement('li');
+            var checkbox = document.createElement('input');
+            var p = document.createElement('p');
+            var delButton = document.createElement('button');
+
+            checkbox.type = 'checkbox';
+            checkbox.id = `c${i}`;
+            checkbox.addEventListener("click", checked);
+            if (itemsCopy[i].status == "checked") {
+                checkbox.checked = true;
+            }
+
+            p.id = `p${i}`;
+            p.innerText = itemsCopy[i].text;
+
+            delButton.type = 'button';
+            delButton.className = 'button';
+            var del = document.createElement('i');
+            del.className = 'fa fa-trash';
+            delButton.appendChild(del);
```

Comment:
You can add an event listener to the font awesome trash icon itself no need to use button and also appending the Icon inside it.